src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/Token.java

```diff
@@ -65,6 +66,9 @@ final class Token implements Serializable {
     @SuppressWarnings("serial") // Type of field is not Serializable
     final Config config;
 
+    @SuppressWarnings("serial") // Type of field is not Serializable
+    final CTSVariant ctsVariant;
```

Comment:
For new non-serializable fields, you should just mark it with "transient" keyword and no need for the `@SuppressWarnings("serial")`

Reply:
Ok, done in 476cbc97f109ae2507706f1b74b71581207fdc99.